integration/kubernetes/init.sh

```diff
@@ -40,6 +40,33 @@ wait_pods_ready()
 	done
 }
 
+build_custom_stress_image()
+{
+	info "Build custom stress image"
+	image_version=$(get_test_version "docker_images.registry.version")
+	registry_image=$(get_test_version "docker_images.registry.url" | cut -d '/' -f 5-):"${image_version}"
```

Comment:
@devmic, we can get the image name from the URL in `versions.yaml` file. I updated it, can you PTAL?